module/proto/gravity/v2/query.proto

```diff
@@ -0,0 +1,371 @@
+syntax = "proto3";
+package gravity.v2;
+
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "gravity/v2/genesis.proto";
+import "gravity/v2/gravity.proto";
+import "gravity/v2/msgs.proto";
+
+option go_package = "github.com/peggyjv/gravity-bridge/module/v3/x/gravity/types";
+
+// Query defines the gRPC querier service
+service Query {
+
+  // Module parameters query
+  rpc Params(ParamsRequest) returns (ParamsResponse) {
+    // option (google.api.http).get = "/gravity/v1/params";
```

Comment:
Commented out, they do nothing. They are to generate rest endpoints for usage. Clearly we don't need them, since they are commented out. I would rather remove them if its an option

Reply:
If they're not doing anything we should probably delete them, but I am interested in how this module is handling these paths in their absence.

Reply:
My guess is that it isn't. It is probably using rpc only